app/views/idv/address/new.html.erb

```diff
@@ -26,13 +26,15 @@
     <%= f.input :city, label: t('idv.form.city'), required: true, maxlength: 255,
                        input_html: { aria: { invalid: false }, value: @pii['city'] } %>
 
-    <div class="clearfix margin-x-neg-1">
-      <div class="sm-col sm-col-8 padding-x-1">
+    <div class="margin-x-neg-1">
```

Comment:
It seems like these `margin-x-neg-1` and `padding-x-1` wrappers may only have been necessary to support the two fields being shown in the same line, so we might be able to remove them now?